src/local.js

```diff
@@ -46,6 +47,24 @@ function cachedToken(scope) {
 		});
 }
 
+function adjustClockSkew(res) {
+	var dateHeader = res.headers && res.headers.date;
+	if (!dateHeader) {
+		return;
+	}
+
+	var serverTime = new Date(dateHeader).getTime();
```

Comment:
This is apparently not the best way to go about it (date string parsing is somewhat inconsistent across browsers, according to MDN - but I think it should be sufficient for us right now.